packages/webapp/src/inductions/components/induction-lists/invitee-inductions.tsx

```diff
@@ -0,0 +1,105 @@
+import dayjs from "dayjs";
+import * as relativeTime from "dayjs/plugin/relativeTime";
+
+import { getEndorsementsByInductionId } from "inductions/api";
+import { getInductionStatus } from "inductions/utils";
+import { useFetchedData } from "_app";
+import * as InductionTable from "inductions/components/induction-lists/induction-table";
+import { Endorsement, Induction, InductionStatus } from "../../interfaces";
+import { InductionActionButton } from "./action-button";
+
+dayjs.extend(relativeTime.default);
+
+interface Props {
+    inductions: Induction[];
+}
+
+export const InviteeInductions = ({ inductions }: Props) => (
+    <InductionTable.Table
+        columns={INVITEE_INDUCTION_COLUMNS}
+        data={getTableData(inductions)}
+        tableHeader="My invitations to Eden"
+    />
+);
+
+const INVITEE_INDUCTION_COLUMNS: InductionTable.Column[] = [
+    {
+        key: "inviter",
+        label: "Inviter",
+    },
+    {
+        key: "voters",
+        label: "Voters",
+        className: "hidden md:flex",
+    },
+    {
+        key: "time_remaining",
+        label: "Time remaining",
+        className: "hidden md:flex",
+    },
+    {
+        key: "status",
+        label: "Action/Status",
+        type: InductionTable.DataTypeEnum.Action,
+    },
+];
+
+const getTableData = (inductions: Induction[]): InductionTable.Row[] => {
+    return inductions.map((ind) => {
+        const [allEndorsements] = useFetchedData<any>(
```

Comment:
This is very interesting. I noticed that you did that in the above endorsements table too, but I never used a hook inside a function. I usually declare the hooks in the component and then I reuse the properties. Does it behave well with componentes re-rendering, updates and attributes changes? If yes, this is pretty cool! 🤯

Reply:
Yes, in this case it does behave reliably, but only because it's getting called in the same order with every render, it doesn't interact with state and the interplay with other hooks, _and_ I'm not calling it conditionally anywhere. 

That said, keeping hooks at the top level in a component _is_ a best practice, and best practices are best practices for a reason. If it's nested inside a function, I or others may be tempted to try to call that function conditionally in the future. Or it may later become dependent on other hooks in ways that complicate it.

I'm going to go ahead and move these to the top level. Good call.

Reply:
On second thought, I'm leaving this as is for now. Order of execution is guaranteed and these are simple implementations.